### DISCUSSION TITLE
regarding the suggestions to use a different editor

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#222] [Admin General]
i want the advanced editor to be written by us, for bark. that way we can optimize it to suit our needs in the best possible way. we would also understand it more than other options.

that's why i'm so apprehensive about a different editor, not written by us

**Comment by mariocraft987** [comment]
k

**Comment by raynepaws** [comment]
i've received more and more editor recommendations. if you guys don't want my editor we don't have to use it, i can make my own repository for it. but i'm just saying i think it's best to make our own editor (after seeing the catastrophe that is the block editor)

**Reply by dumorando** [reply]
the block editor should be rewritten from the ground up imo

**Reply by raynepaws** [reply]
i agree
